### PR TITLE
Feature/303 orderbook improvements

### PR DESCRIPTION
**Pull request by bglownia**

### CODE REVIEW

**Review by MadalinaRaicu — APPROVED** [review]

File: libs/market-depth/src/lib/orderbook.module.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  width: 6px;
+  background-color: #999;
+}
+
+.scroll::-webkit-scrollbar-thumb {
+  width: 6px;
+  background-color: #333;
+}
+.scroll::-webkit-scrollbar-track {
+  box-shadow: inset 0 0 6px rgb(0 0 0 / 30%);
```
Comment (sam-keen) [inline libs/market-depth/src/lib/orderbook.module.scss:18]: Chore (blocking): Worth putting this '6px' value in a var.

File: libs/market-depth/src/lib/orderbook.spec.tsx [hunk 2]
```diff
@@ -1,10 +1,165 @@
...
+  it('should scroll to mid price on init', async () => {
+    window.innerHeight = 11 * rowHeight;
+    const result = render(
+      <Orderbook
+        decimalPlaces={decimalPlaces}
+        {...generateMockData(params)}
+        onResolutionChange={onResolutionChange}
+      />
+    );
+    await waitFor(() => screen.getByTestId('bid-vol-122900'));
```
Comment (sam-keen) [inline libs/market-depth/src/lib/orderbook.spec.tsx:29]: Chore (blocking): used in lots of places:

```suggestion
    await waitFor(() => screen.getByTestId(`bid-vol-${params.midPrice}`));
```

File: libs/react-helpers/src/lib/grid/vol-cell.tsx [hunk 3]
```diff
@@ -15,23 +21,28 @@ export interface IVolCellProps extends ICellRendererParams {
...
+    }
+    return (
+      <div className="relative" data-testid={testId || 'vol'}>
+        <div
+          className={classNames('h-full absolute top-0', {
+            'left-0': type === VolumeType.bid,
+            'right-0': type === VolumeType.ask,
+          })}
+          style={{
+            width: relativeValue ? `${relativeValue}%` : '0%',
```
Comment (sam-keen) [inline libs/react-helpers/src/lib/grid/vol-cell.tsx:37]: Question (non-blocking): why use a style prop instead of classes here?

Comment (bglownia) [reply libs/react-helpers/src/lib/grid/vol-cell.tsx:37]: color styles will be changed to classes in another PR

**Review by sam-keen — APPROVED** [review]

**Pull request merged**